x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/util.ts

```diff
@@ -48,3 +50,45 @@ export function mergeAndReplaceCategoryCounts(
 
   return merged;
 }
+
+export function findReplacementsForEprPackage(
+  replacements: CustomIntegration[],
+  packageName: string,
+  category: Category,
+  release: 'beta' | 'experimental' | 'ga'
+): CustomIntegration[] {
+  if (release === 'ga') {
+    return [];
+  }
+
+  const filtered = replacements.filter((customIntegration: CustomIntegration) => {
+    return (
+      customIntegration.eprOverlap === packageName &&
+      (!category || customIntegration.categories.includes(category))
+    );
+  });
+  return filtered;
+}
+
+export function mergeEprPackagesWithReplacements(
```

Comment:
Either we should extract a hook that gets tested as I mentioned in the `AvailablePackages` component, or at the very least we should unit test this function.

Reply:
we actually would need access to this functinality to show the corresponding integrations in the EPR-package overview page https://github.com/elastic/kibana/blob/4681a80317176543535bfdb2188d94aa40d741fc/x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/overview/overview.tsx#L29. We were going to split-off that display in separate PR.

That said, this export is fluffy in this context, so can remove it.